week-4/Homework/mandatory/2-writers.js

```diff
@@ -63,10 +65,21 @@ Exercise 2:
   "Writer {firstName} {lastName} died at {age} years old."
 */
 
+for (let writer of writers) {
+    if (50 > writer.age && writer.age >= 40 && writer.alive === false) {

```

Comment:
so alive is a boolean itself
so do you need to compare it witha boolean value
I mean if you checking with false value
!writer.alive will be enough
``` if (50 > writer.age && writer.age >= 40 && !writer.alive )```

Reply:
Also, try to follow the same practice all the time
writer.age<, writer age>

Reply:
done